index.html

```diff
@@ -4,10 +4,55 @@
   <meta charset="utf-8">
   <meta name="Description" content="INF 7420 Project">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Document</title>
+  <title>Jim Cheng - Portfolio</title>
 </head>
 <body>
-  <h1>Hello World!</h1>
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/_a0GdDydowc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <header>
+
+    <nav>
+
+      <img src="images/yourlogohere.png" alt=" company logo ONE">
+      <img src="images/company name.jpg" alt=" company name TWO">
+       <ul>
+        <li> <a href="home.html">Home</a></li>
+        <li><a href="about us.html">About us</a></li>
+        <li><a href="specialist area.html">Specialist Area</a></li>
+        <li><a href="claims.html">Claims</a></li>
+        <li><a href="contact us.html">Contact us</a></li>
```

Comment:
try to avoid spaces. use dashes instead. change file to contact-us.html and edit this href.